src/analyzer/analyzergain.h

```diff
@@ -15,18 +15,18 @@ class ReplayGain;
 
 class AnalyzerGain : public Analyzer {
```

Comment:
I would like to keep it separate, because it will be easier to though out the RG1 code some day. 
I also think this separation improved readability.